ckb_syscalls.h

```diff
@@ -377,6 +377,47 @@ int ckb_exec_cell(const uint8_t* code_hash, uint8_t hash_type, uint32_t offset,
                  argv);
 }
 
+typedef struct {
```

Comment:
What about exposing this data structure directly? In general having too many arguments(I'm counting 9 or 10 below) is not a good thing.

Reply:
I redesigned the API, now it takes fewer parameters